### ISSUE TITLE
Left and Right Braces do not trigger onChange event

### ISSUE THREAD

**Issue by babaskate** [hodgef/simple-keyboard#12]
I'm noticing this in the demo as well, but when you type a left or right brace, the onChange callback does not fire. We have a keyboard config setup such as this: 
```
/*
              * u20AC=€ (euro) u00A3=£ (pound)
              * u00A5=¥ (yen) u00A7=§ (section [for law])
              * u007B == {  u007D == } 
*/
 symbols: [
            '@gmail.com @yahoo.com @outlook.com @hotmail.com @mail.com @aol.com',
            '1 2 3 4 5 6 7 8 9 0 {bksp}',
            '\u20AC \u00A3 \u00A5 _ ^ [ ] \u007B \u007D {enter}',
            '{numbers} \u00A7 | ~ ... \\ < > ! ? {numbers}',
            '{abc} .com {space} {abc}'
        ]
```
Only the button press event fires when you type either the left or right brace, so in that callback, I explicitly check for the values, and reset the simple keyboard input and the value of my text box. 
```
if (button === '\u007B' || button === '\u007D') {
    const newValue = `${keyboard.getInput()}${button}`;
    keyboard.setInput(newValue);
    emailInput.setValue(newValue);
}
```

The above solution works for my purposes, but I'm assuming this is a bug in the library. I'll continue to investigate to see where there's an issue.

**Comment by hodgef** [comment]
Hello @babaskate,

Sorry, I'm not too sure what you're meaning to achieve and the issue.
Could you please edit/fork the following example to show your use-case and send me the link?

[![Edit simple-keyboard demo - npm](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/krzkx19rr)

I'm also available on the chat:
http://franciscohodge.com/simple-keyboard/chat/join

Regards

**Comment by hodgef** [comment]
Hello @babaskate,

Thank so much for this bug report. I was able to reproduce the issue and it is now fixed as of 2.5.8.

[![Edit simple-keyboard demo - npm](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/krzkx19rr?module=%2Fsrc%2Findex.js)

If you still have the issue, please let me know and I will reopen the ticket. Thanks!
